### PR TITLE
PIM-5363: update the catalogs to add the last export/import job

### PR DESCRIPTION
**Pull request by arnolanglade**
It adds the last import/export jobs to every catalogs. 

FYI : The PR seems huge but it just removes indentation (it is not manage by the command...)

### CODE REVIEW

File: 1.6/community/large/fixtures/jobs.yml [hunk 1]
```diff
@@ -1,262 +1,248 @@
 jobs:
     csv_product_import:
-        connector: Akeneo CSV Connector
-        alias:     csv_product_import
-        label:     Demo product import
-        type:      import
+        connector: 'Akeneo CSV Connector'
+        alias: csv_product_import
+        label: 'Product import'
```
Comment (pierallard) [inline 1.6/community/large/fixtures/jobs.yml]: Csv product import, no?

File: 1.6/community/large/fixtures/jobs.yml [hunk 2]
```diff
@@ -1,262 +1,248 @@
...
+            delimiter: ;
+            enclosure: '"'
+            escape: \
+            enabled: true
+            categoriesColumn: categories
+            familyColumn: family
+            groupsColumn: groups
             realTimeVersioning: true
-            decimalSeparator: .
-            dateFormat: yyyy-MM-dd
```
Comment (pierallard) [inline 1.6/community/large/fixtures/jobs.yml:19]: Please check you can always save your job without this line. I think there is an error in the PIM.

File: 1.6/enterprise/large/fixtures/job_profile_accesses.csv [hunk 3]
```diff
@@ -20,19 +20,33 @@ asset_import;"IT support,Manager";"IT support,Manager"
...
+xlsx_category_import;"IT support,Manager";"IT support,Manager"
+xlsx_category_export;"IT support,Manager";"IT support,Manager"
+xlsx_association_type_import;"IT support,Manager";"IT support,Manager"
+xlsx_association_type_export;"IT support,Manager";"IT support,Manager"
+xlsx_group_import;"IT support,Manager";"IT support,Manager"
+xlsx_group_export;"IT support,Manager";"IT support,Manager"
+xlsx_attribute_import;"IT support,Manager";"IT support,Manager"
+xlsx_attribute_export;"IT support,Manager";"IT support,Manager"
+xlsx_option_import;"IT support,Manager";"IT support,Manager"
+xlsx_option_export;"IT support,Manager";"IT support,Manager"
```
Comment (pierallard) [inline 1.6/enterprise/large/fixtures/job_profile_accesses.csv]: "csv_product_import_with_rules" ? "xlsx_product_import_with_rules" ?

File: 1.6/enterprise/large/fixtures/jobs.yml [hunk 4]
```diff
@@ -181,6 +184,45 @@ jobs:
...
+            enclosure: '"'
+            escape: \
+            enabled: true
+            categoriesColumn: categories
+            familyColumn: family
+            groupsColumn: groups
+            realTimeVersioning: true
+            decimalSeparator: .
+            dateFormat: yyyy-MM-dd
+    xlsx_clothing_product_import_with_rules:
```
Comment (pierallard) [inline 1.6/enterprise/large/fixtures/jobs.yml]: clothing

File: 1.6/enterprise/large/fixtures/jobs.yml [hunk 5]
```diff
@@ -317,3 +359,87 @@ jobs:
...
             withHeader: true
             filePath: /tmp/attribute-option.csv
+    xlsx_product_quick_export:
+        connector: 'Akeneo Mass Edit Connector'
+        alias: xlsx_product_quick_export
+        label: 'XLSX product quick export'
+        type: quick_export
+        configuration:
+            withHeader: true
+            filePath: /tmp/products_export_%locale%_%scope%_%datetime%.xlsx
```
Comment (pierallard) [inline 1.6/enterprise/large/fixtures/jobs.yml]: Why all these jobs are only in EE ?

File: 1.6/enterprise/large/fixtures/jobs.yml [hunk 6]
```diff
@@ -317,3 +359,87 @@ jobs:
             enclosure: '"'
             withHeader: true
             filePath: /tmp/attribute-option.csv
+    xlsx_product_quick_export:
+        connector: 'Akeneo Mass Edit Connector'
```
Comment (pierallard) [inline 1.6/enterprise/large/fixtures/jobs.yml]: xlsx_quick_export uses Akeneo Mass Edit Connector ? (don't know, it's a question)

File: 1.6/enterprise/small/fixtures/attribute_options.csv [hunk 7]
```diff
@@ -29,263 +29,203 @@ attr_8;attr_opt_attr_86;8;"Voluptas dicta quisquam."
 attr_8;attr_opt_attr_87;7;"Libero voluptas ut."
 attr_8;attr_opt_attr_88;5;"Rerum quo perferendis."
 attr_8;attr_opt_attr_89;6;"Accusamus dolorem."
-attr_9;attr_opt_attr_90;1;"Soluta modi."
```
Comment (pierallard) [inline 1.6/enterprise/small/fixtures/attribute_options.csv:32]: Whaaaat?

File: 1.6/enterprise/small/fixtures/attributes.csv [hunk 8]
```diff
@@ -1,103 +1,103 @@
 code;type;group;useable_as_grid_filter;label-en_US;localizable;scopable;allowed_extensions;metric_family;default_metric_unit
-sku;pim_catalog_identifier;attr_gr_1;1;"identifier dignissimos qui";;;;;
```
Comment (pierallard) [inline 1.6/enterprise/small/fixtures/attributes.csv:2]: Whaaaaat ?

File: 1.6/enterprise/small/fixtures/families.csv [hunk 9]
```diff
@@ -1,51 +1,51 @@
 code;label-en_US;attribute_as_label;attributes;requirements-ecommerce;requirements-print
-fam_0;"Sed perferendis.";name;sku,name,attr_8,attr_82,attr_93,attr_89,attr_64,attr_19,attr_68,attr_26,attr_31,attr_51,attr_29,attr_81,attr_7,attr_45,attr_99,attr_50,attr_55,attr_17,attr_59,attr_75,attr_32,attr_40,attr_49,attr_66,attr_84,attr_10,attr_78,attr_63,attr_33,attr_28,attr_30,attr_3,attr_94,attr_97,attr_1,attr_2,attr_47,attr_39,attr_74,attr_18,attr_57,attr_58,attr_91,attr_52,attr_77,attr_69,attr_73,attr_23,attr_4;sku,attr_30,attr_64,attr_73,attr_31;sku,attr_97,attr_7,attr_81,attr_39
```
Comment (pierallard) [inline 1.6/enterprise/small/fixtures/families.csv:2]: Whaaaaaaaaaaaaaaaaaaaaaaat ?

File: 1.6/enterprise/small/fixtures/variant_groups.csv [hunk 10]
```diff
@@ -1,301 +1,301 @@
-code;axis;type;label-en_US;attr_0;attr_89;attr_58;attr_3;attr_30;attr_66;attr_99;attr_23;name
```
Comment (pierallard) [inline 1.6/enterprise/small/fixtures/variant_groups.csv:1]: Ohh god

File: 1.6/enterprise/large/fixtures/jobs.yml (hunk 4 above)
Comment (arnolanglade) [inline 1.6/enterprise/large/fixtures/jobs.yml]: done

**Comment by pierallard** [comment]
GTM

**Pull request merged**